added links to Emily's MTH 95 videos for all sections

aecary opened the pull request.

Alex-Jordan commented:
We've moved to doing work on master if it doesn't impact numbering or other important things. Could the video work be done on master?

If so, note the hashes for these commits here. Then switch to your master and pull to update it. Then use git cherry-pick to apply each commit by its hash. After it's all in master, push and open a new PR. At some point you will have to delete edition on your GH account. Locally, you can use git reset HEAD~5 to bring edition back to where it was. Then push edition to GH to get it back there.

aecary commented:
These don't change Figure numbers, so all should go to master. Sorry I
forgotten we'd switched.

I wasn't confident I could follow those steps, and I was able to get
further with the updates to the MTH 60/65 lists than I was yesterday. I
switched to master, pulled, and updated the playlists there. I pushed those
to master, and there's an open pull request.  I think all that needs to
happen now is that this commit is deleted or reverted. Is that something
you can do? If not, please advise on how to do it.

-Ann

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sat, Sep 1, 2018 at 10:26 AM, Alex Jordan <notifications@github.com>
wrote:

> We've moved to doing work on master if it doesn't impact numbering or
> other important things. Could the video work be done on master?
>
> If so, note the hashes for these commits here. Then switch to your master
> and pull to update it. Then use git cherry-pick to apply each commit by its
> hash. After it's all in master, push and open a new PR. At some point you
> will have to delete edition on your GH account. Locally, you can use git
> reset HEAD~5 to bring edition back to where it was. Then push edition to GH
> to get it back there.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417874742>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwA1VCpvPyelMNhW-csX5bDgNfY6xLks5uWsM5gaJpZM4WWE8E>
> .
>

Alex-Jordan commented:
The commit exists on your computer's edition branch and on your GH's
repository's edition branch. Those are both places I can't directly do
anything with. And as long as they are there, you will have trouble pulling
to your edition branch from the PCC GH repo's edition branch.

There were 5 commits here, but only 3 on the PR in master. Are you sure
everything in the 5 commits can be cleared away? If so, for your local:

git checkout edition
git reset HEAD~5
git status
git checkout src

for GH, delete the edition branch after clicking "branches". Then push to
edition from your local.



On Sat, Sep 1, 2018 at 11:13 PM, aecary <notifications@github.com> wrote:

> These don't change Figure numbers, so all should go to master. Sorry I
> forgotten we'd switched.
>
> I wasn't confident I could follow those steps, and I was able to get
> further with the updates to the MTH 60/65 lists than I was yesterday. I
> switched to master, pulled, and updated the playlists there. I pushed those
> to master, and there's an open pull request. I think all that needs to
> happen now is that this commit is deleted or reverted. Is that something
> you can do? If not, please advise on how to do it.
>
> -Ann
>
> *Ann Cary*
> (she/her pronouns)
> Mathematics Instructor
> Curriculum Committee Chair
> Portland Community College
> Sylvania Campus, ST 203
> 971-722-7590
> <http://goog_233805033>
> http://spot.pcc.edu/~acary/
>
>
>
> On Sat, Sep 1, 2018 at 10:26 AM, Alex Jordan <notifications@github.com>
> wrote:
>
> > We've moved to doing work on master if it doesn't impact numbering or
> > other important things. Could the video work be done on master?
> >
> > If so, note the hashes for these commits here. Then switch to your master
> > and pull to update it. Then use git cherry-pick to apply each commit by
> its
> > hash. After it's all in master, push and open a new PR. At some point you
> > will have to delete edition on your GH account. Locally, you can use git
> > reset HEAD~5 to bring edition back to where it was. Then push edition to
> GH
> > to get it back there.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417874742>,
> or mute
> > the thread
> > <https://github.com/notifications/unsubscribe-auth/AGBwA1VCpvPyelMNhW-
> csX5bDgNfY6xLks5uWsM5gaJpZM4WWE8E>
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417907809>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AEg3AMVOk_ti4vaRflkbD1okzqI_ny5Eks5uW3cggaJpZM4WWE8E>
> .
>



-- 
Alex Jordan
Mathematics Instructor
Portland Community College

aecary commented:
The two that didn't carry over were todos about videos that are still
needed. Those aren't the only two, and I created a separate log of notes
elsewhere.

*Ann Cary*
(she/her pronouns)
Mathematics Instructor
Curriculum Committee Chair
Portland Community College
Sylvania Campus, ST 203
971-722-7590
<http://goog_233805033>
http://spot.pcc.edu/~acary/



On Sun, Sep 2, 2018 at 7:30 AM, Alex Jordan <notifications@github.com>
wrote:

> The commit exists on your computer's edition branch and on your GH's
> repository's edition branch. Those are both places I can't directly do
> anything with. And as long as they are there, you will have trouble pulling
> to your edition branch from the PCC GH repo's edition branch.
>
> There were 5 commits here, but only 3 on the PR in master. Are you sure
> everything in the 5 commits can be cleared away? If so, for your local:
>
> git checkout edition
> git reset HEAD~5
> git status
> git checkout src
>
> for GH, delete the edition branch after clicking "branches". Then push to
> edition from your local.
>
>
>
> On Sat, Sep 1, 2018 at 11:13 PM, aecary <notifications@github.com> wrote:
>
> > These don't change Figure numbers, so all should go to master. Sorry I
> > forgotten we'd switched.
> >
> > I wasn't confident I could follow those steps, and I was able to get
> > further with the updates to the MTH 60/65 lists than I was yesterday. I
> > switched to master, pulled, and updated the playlists there. I pushed
> those
> > to master, and there's an open pull request. I think all that needs to
> > happen now is that this commit is deleted or reverted. Is that something
> > you can do? If not, please advise on how to do it.
> >
> > -Ann
> >
> > *Ann Cary*
> > (she/her pronouns)
> > Mathematics Instructor
> > Curriculum Committee Chair
> > Portland Community College
> > Sylvania Campus, ST 203
> > 971-722-7590
> > <http://goog_233805033>
> > http://spot.pcc.edu/~acary/
> >
> >
> >
> > On Sat, Sep 1, 2018 at 10:26 AM, Alex Jordan <notifications@github.com>
> > wrote:
> >
> > > We've moved to doing work on master if it doesn't impact numbering or
> > > other important things. Could the video work be done on master?
> > >
> > > If so, note the hashes for these commits here. Then switch to your
> master
> > > and pull to update it. Then use git cherry-pick to apply each commit by
> > its
> > > hash. After it's all in master, push and open a new PR. At some point
> you
> > > will have to delete edition on your GH account. Locally, you can use
> git
> > > reset HEAD~5 to bring edition back to where it was. Then push edition
> to
> > GH
> > > to get it back there.
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417874742>,
> > or mute
> > > the thread
> > > <https://github.com/notifications/unsubscribe-auth/AGBwA1VCpvPyelMNhW-
> > csX5bDgNfY6xLks5uWsM5gaJpZM4WWE8E>
> > > .
> > >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417907809>,
> or mute
> > the thread
> > <https://github.com/notifications/unsubscribe-auth/AEg3AMVOk_
> ti4vaRflkbD1okzqI_ny5Eks5uW3cggaJpZM4WWE8E>
> > .
> >
>
>
>
> --
> Alex Jordan
> Mathematics Instructor
> Portland Community College
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PCCMathSAC/orcca/pull/454#issuecomment-417934816>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AGBwA7SSEV1MzJrnY4I3gVnxfbaECFJiks5uW-uUgaJpZM4WWE8E>
> .
>
